packages/skygear-core/src/container.ts

```diff
@@ -27,6 +36,35 @@ export class AuthContainer<T extends BaseAPIClient> {
     this.currentSessionID = null;
   }
 
+  async saveExtraSessionInfoOptions() {
```

Comment:
Why do we need to save this?

Reply:
As mentioned in design, the developer can set the device name once, and the SDK will use the same device name on every app launch.